### PR TITLE
Add CSS Handle to carousel dot

### PR DESCRIPTION
**Pull request by kaisermann**
#### What problem is this solving?

Add missing css handle to pagination dot

Related to https://app.clubhouse.io/vtex/story/36117/css-handles-on-dotscontainer

#### How should this be manually tested?

1) https://kiwi--storecomponents.myvtex.com/jumper-best-friend/p
2) Do a `document.querySelectorAll('.vtex-store-components-3-x-productImagesGallerySwiperDot')` and check if the result has some nodes.

#### Checklist/Reminders

- [x] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

### CODE REVIEW

File: react/components/ProductImages/components/Carousel/index.js [hunk 1]
```diff
@@ -405,31 +435,43 @@ class Carousel extends Component {
...
+      }
+    )
 
     const SliderComponent = slides.length === 1 ? FakeSwiper : Swiper
 
     return (
       <div className={containerClasses} aria-hidden="true">
         {isThumbsVertical && thumbnailSwiper}
         <div className={imageClasses}>
+          <div>potato</div>
```
Comment (klzns) [inline react/components/ProductImages/components/Carousel/index.js:467]: Ops
```suggestion
```

File: react/components/ProductImages/components/Carousel/index.js [hunk 2]
```diff
@@ -225,13 +246,14 @@ class Carousel extends Component {
...
-          clickable: true,
-          bulletActiveClass: 'c-action-primary swiper-pagination-bullet-active',
-        },
-      }),
+      ...(slides.length > 1 &&
+        showPaginationDots && {
+          pagination: {
+            el: '.swiper-pagination',
+            clickable: true,
+            bulletActiveClass: `${cssHandles.productImagesGallerySwiperDot} c-action-primary swiper-pagination-bullet-active`,
```
Comment (Klynger) [inline react/components/ProductImages/components/Carousel/index.js:254]: Why only in the active element?

File: react/components/ProductImages/components/Carousel/index.js (hunk 1 above)
Comment (kaisermann) [reply react/components/ProductImages/components/Carousel/index.js:467]: oooooooops

**Comment by kaisermann** [comment]
It's not possible to customize the `bulletClass` in the same manner we customized the `bulletActiveClass`. While `bulletActiveClass` is only used to add or remove the class itself, the `bulletClass` is also used as a selector to find the bullet elements. That means that using a `bulletClass` with a value like `swiper-pagination-bullet vtex-bullet`, the bullets will stop working because the `swiper` library will try to find elements with the selector: `.swiper-pagination-bullet vtex-bullet` (note the period at the start of the selector). 

I tried using `patch-package` to fix it, but `builder-hub` doesn't run npm scripts for security reasons. 

A possible solution would be to fork `swiper` AND `react-id-swiper` and fix the code ourselves. However, I'm strictly against this for maintainability reasons. 

Another solution would be to find another swipe library and replace `react-id-swiper`.

**Pull request closed without merging**